site/public/css/server.css

```diff
@@ -1749,6 +1748,10 @@ end of styles used in the admin gradeable page
     z-index: 999;
 }
 
+#skip-nav {
+    margin: 90px 10px 10px; /* 90px for header */
```

Comment:
This comment isn't very descriptive, something like "make skip to main content box" larger
would be more helpful if you're going to add a comment here.